Formula/nfdump.rb

```diff
@@ -1,8 +1,9 @@
 class Nfdump < Formula
-  desc "Tools to collect and process netflow data on the command line"
+  desc "Tools to collect and process netflow data on the command-line"
   homepage "http://nfdump.sourceforge.net"
-  url "https://downloads.sourceforge.net/project/nfdump/stable/nfdump-1.6.13/nfdump-1.6.13.tar.gz"
-  sha256 "251533c316c9fe595312f477cdb051e9c667517f49fb7ac5b432495730e45693"
+  url "https://github.com/phaag/nfdump.git",
```

Comment:
We prefer to use the tarball. It's a slightly quicker operation than a git checkout, and it can be protected with a stronger checksum than git's native SHA1.